app/components/forms/events/view/order-form.js

```diff
@@ -20,11 +23,47 @@ export default Component.extend(FormMixin, {
       }
     };
   },
+  typeList: computed(function() {
+    return orderBy(fieldTypes, 'type');
+  }),
+  normalFields: computed('data.customForms', function() {
```

Comment:
You're using `isCustomQuestion`. So your computed property should be watching `data.customForms.@each.isCustomQuestion`